src/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -175,7 +175,7 @@ export const PaymentContent: FC<Props> = props => {
         <Spacer mb={2} />
         <BankDebitProvider
           order={order}
-          paymentMethod={paymentMethod}
+          paymentMethod="SEPA_DEBIT"
```

Comment:
It seems more reasonable to pass in the specific payment method; otherwise, we'll render multiple `Collapse`s with the same payment method. Let me know if I misunderstood things here!